src/init/setup/SetupHttpHandler.ts

```diff
@@ -33,6 +33,11 @@ export interface SetupHttpHandlerArgs {
    * Renders the main view.
    */
   templateEngine: TemplateEngine;
+  /**
+   * Determines if data can be written to the root of the server.
+   * Relevant to make sure there are no nested storages if registration is enabled for example.
+   */
+  disableRoot?: boolean;
```

Comment:
```suggestion
  disableRootWrite?: boolean;
```

Reply:
(ideally `allowRootWrite` but that gets complicated with it being optional)

Reply:
Could have it be optional and default to `true` so should be fine to change I think.